src/CanvasRenderingContext2d.cc

```diff
@@ -2684,8 +2692,10 @@ NAN_METHOD(Context2d::MeasureText) {
       x_offset = 0.0;
   }
 
+  // are those two line useful?
   cairo_matrix_t matrix;
   cairo_get_matrix(ctx, &matrix);
+
```

Comment:
matrix does not seem to be used in the function. Is `cairo_get_matrix` having some side effect that is necessary for the code below to work?

Reply:
It looks like it is used for `alphabeticBaseline` below